.github/header-checker-lint.yml

```diff
@@ -18,7 +18,7 @@ ignoreFiles:
   - "**/ghcnd-stations.txt"
   - "texttospeech/snippets/resources/example.txt"
   - "texttospeech/snippets/resources/hello.txt"
-
+  - "automl/snippets/resources/input.txt"
```

Comment:
Would it be possible to instead try and add a header? `python-automl` didn't seem like it added a header nor got approval to add them into ignoring them. If not, fine to leave it here.

Reply:
Yeah, it should be fine.  It's a 21 byte text file used in a single test (not really source code per se.) But that test doesn't even depend on the contents of the file (i.e. it will pass no matter what is in there, as far as I can tell.)

Refactoring automl snippet tests is a little out of my bailiwick, so I'll just add the header and move on.